src/views/OfferInfoPage/components/DesignerInfoSection.tsx

```diff
@@ -1,35 +1,38 @@
 import { styled } from 'styled-components';
 
 import { IcLeft } from '../assets/icons';
-import { OFFER_DATA } from '../constants/OFFER_DATA';
+import useGetOfferModel from '../hooks/useGetOfferModel';
 
 const DesignerInfoSection = () => {
-  const DesingerInfo = OFFER_DATA.data.designerInfo;
+  const { data } = useGetOfferModel();
+
+  const DesingerInfo = data?.designerInfo;
```

Comment:
```suggestion
  const { imgUrl, shopName, name, instagramUrl ... } = data?.designerInfo;
```
이렇게 구조분해할당 해주면 DesignerInfo.~ 이렇게 일일이 안해줘도 된답니당!! 
DesignerInfo가 없을 때의 예외처리를 해주고 싶다면 매번 옵셔널체이닝 해주지 않고 최상단에 DesignerInfo && 이렇게 조건부 렌더링 해줘도 된답니당